packages/sdk/contractkit/src/wrappers/Accounts.ts

```diff
@@ -203,18 +200,22 @@ export class AccountsWrapper extends BaseWrapper<Accounts> {
     proofOfSigningKeyPossession: Signature
   ): Promise<CeloTransactionObject<void>> {
     const validators = await this.kit.contracts.getValidators()
-    const account = this.kit.defaultAccount || (await this.kit.web3.eth.getAccounts())[0]
+    const account =
+      this.kit.connection.defaultAccount || (await this.kit.connection.getAccounts())[0]
     if (await validators.isValidator(account)) {
-      const message = this.kit.web3.utils.soliditySha3({ type: 'address', value: account })
-      const prefixedMsg = hashMessageWithPrefix(message)
+      const message = this.kit.connection.web3.utils.soliditySha3({
+        type: 'address',
+        value: account,
+      })
+      const prefixedMsg = hashMessageWithPrefix(message!)
```

Comment:
why did these become optionals?

Reply:
Web3Utils can return a null, and we use it in hashMessageWithPrefix.
I've just changed it to force the utils to return a string